description.ext

```diff
@@ -1,4 +1,13 @@
 // ============================================================================================
+// Loading Screen
+
+onLoadName = "*** Insert Mission Name Here. ***";

```

Comment:
Everything but onLoadName is specified already in the editor/mission.sfm and shouldn't be overwritten by description.ext.